File: RDCPanel/RDCPanel.Web/Modules/Employees/Employee/EmployeeColumns.cs
using Serenity.ComponentModel;
using System.ComponentModel;

namespace RDCPanel.Employees.Columns;

[ColumnsScript("Employees.Employee")]
[BasedOnRow(typeof(EmployeeRow), CheckNames = true)]
public class EmployeeColumns
{
    [EditLink, DisplayName("Db.Shared.RecordId"), AlignRight]
    public int Id { get; set; }
    [EditLink]
    public string Name { get; set; }
    public string JobTitle { get; set; }
    public string Department { get; set; }
    public string Extension { get; set; }
    public string Email { get; set; }
    public string Image { get; set; }
    public string Branch { get; set; }
    public string Initial { get; set; }
}

File: RDCPanel/RDCPanel.Web/Modules/ServerTypes/Employees/EmployeeColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { EmployeeRow } from "./EmployeeRow";

export interface EmployeeColumns {
    Id: Column<EmployeeRow>;
    Name: Column<EmployeeRow>;
    JobTitle: Column<EmployeeRow>;
    Department: Column<EmployeeRow>;
    Extension: Column<EmployeeRow>;
    Email: Column<EmployeeRow>;
    Image: Column<EmployeeRow>;
    Branch: Column<EmployeeRow>;
    Initial: Column<EmployeeRow>;
}

export class EmployeeColumns extends ColumnsBase<EmployeeRow> {
    static readonly columnsKey = 'Employees.Employee';
    static readonly Fields = fieldsProxy<EmployeeColumns>();
}